protos/src/inventory/v1/product.proto

```diff
@@ -4,24 +4,36 @@ package inventory.v1;
 
 import "buf/validate/validate.proto";
 import "google/protobuf/timestamp.proto";
+import "inventory/v1/category.proto";
+
+enum ProductCategoryStatus {
+  PRODUCT_CATEGORY_STATUS_UNSPECIFIED = 0;
+  /// The product category is active.
+  PRODUCT_CATEGORY_STATUS_ACTIVE = 1;
+  /// The product category is inactive.
+  PRODUCT_CATEGORY_STATUS_INACTIVE = 2;
+}
 
 message ProductCategory {
   // The unique identifier of the product category.
   optional string ref_id = 1;
 
-  // The name of the product category.
-  string name = 2;
+  // The name Internationalized of the product category.
+  inventory.v1.Internationalized name = 2;
+
+  // The status of the product category.
+  ProductCategoryStatus status = 3;
```

Comment:
What is its purpose?

Reply:
It doesn't make sense to have it here; perhaps it belongs in the message category.